### DISCUSSION TITLE
Snapdragon 665

### DISCUSSION THREAD

**Discussion by Mikatakhan** [skyline-emu/skyline#1677] [General]
It would be great, if the developers make the app more optimized for this processor

**Comment by dima-xd** [comment]
Forget about it

**Comment by chostdz** [comment]
Its old GPU and slow potato
You need at least sd 855+

**Reply by Mikatakhan** [reply]
The minimum processor you need for this emulator is sd 660 and 665.so they can easily optimise the emulator for this processor in the future updates

**Reply by Mikatakhan** [reply]
Plus it not a potato processor,if you dont know that,ask anybody even on google

**Reply by JERD001** [reply]
Sd660 has an old GPU that doesn't receive a driver update since quite some time, it's not enough for the emulator. As for sd665 it's capable of booting 2D games but it won't go far. Skyline needs much more horsepower than what those SOCs can offer.

**Reply by Mikatakhan** [reply]
Is adreno 610 enough for this emulator?

**Reply by hope3x** [reply]
No

**Reply by hope3x** [reply]
And who said those are the minimum

**Reply by hope3x** [reply]
845 is the minimum

**Reply by JERD001** [reply]
Not really, maybe in the future for running some 2D games at full speed but not much more

**Reply by JERD001** [reply]
But as hopeless said the minimum the project is pointing at is Adreno 630

**Reply by finr0y** [reply]
Snapdragon845/Adr630 is recommended, along with 8 gigs of ram.
